andru1989/vowels.rb

```diff
@@ -10,14 +10,17 @@ class Vowels
   #   => This method set a Hash to accumulate the number of vowels for each line
   #   => Default: {'a' => 0, 'e' => 0, 'i' => 0, 'o' => 0, 'u' => 0}
   def initialize
-    @vowels = {'a' => 0, 'e' => 0, 'i' => 0, 'o' => 0, 'u' => 0}
+    @vowels   = {'a' => 0, 'e' => 0, 'i' => 0, 'o' => 0, 'u' => 0}
```

Comment:
Unnecessary spacing detected.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.